src/components/MoneyReportHeader.tsx

```diff
@@ -105,7 +124,7 @@ function MoneyReportHeader({
     const isDraft = ReportUtils.isOpenExpenseReport(moneyRequestReport);
     const [isConfirmModalVisible, setIsConfirmModalVisible] = useState(false);
 
-    const transactionIDs = TransactionUtils.getAllReportTransactions(moneyRequestReport?.reportID).map((transaction) => transaction.transactionID);
+    const transactionIDs = TransactionUtils.getAllReportTransactions(moneyRequestReport?.reportID).map((t) => t.transactionID);
```

Comment:
Did the linter complain about this because you're passing `transaction` to the component now? Ideally, we'd use a non-abbreviated variable name here but if it's because of that then I guess it's ok.